### DISCUSSION TITLE
Collapse Table with chakra-ui

### DISCUSSION THREAD

**Discussion by ahmadxgani** [chakra-ui/chakra-ui#7540] [Q&A] [answered]
Hi, I tried to make the table collapse like [this](https://stackblitz.com/edit/nextjs-dtbrys?file=pages/index.js), but it doesn't seem to work as well as [I expected](https://codepen.io/n3k1t/pen/OJMGgyq). and it seems like the "as" property on the collapse component doesn't work. what should i do? is there something i'm missing? any feedback would be greatly appreciated!

I think the https://github.com/chakra-ui/chakra-ui/issues/3051#issuecomment-979410434 is also related to this question as well.

**Comment by aboveyunhai** [comment, marked as answer]
<Collapse /> default `display` style is incorrect for `<tr/>`, you need to override it by inline style. 
```tsx
    <Collapse
      as={Tr}
      in={isOpen}
      style={{ display: 'table-row' }}
      animateOpacity
    ></Collapse
```

**Reply by ahmadxgani** [reply]
Thanks for your help, it works as expected. but I can't seem to add the properties that the tr element has because the collapse motion only accepts motion elements like motion.tr, any suggestions?

**Reply by aboveyunhai** [reply]
> Thanks for your help, it works as expected. but I can't seem to add the properties that the tr element has because the collapse motion only accepts motion elements like motion.tr, any suggestions?

Not sure what do you mean by `tr` property. motion.tr should be able to accept any tr styles and attributes. Sounds like a frame motion question. If you want to be customizable, maybe it's not a good idea to use collapse, just create your own motion.tr instead.
check this: https://chakra-ui.com/getting-started/with-framer
